draft-ietf-core-senml-etch.md

```diff
@@ -252,10 +263,10 @@ Required parameters: none
 Optional parameters: none
 
 Encoding considerations: Must be encoded as using a subset of the
-encoding allowed in {{!RFC8259}}. See RFC-AAAA for details. This
-simplifies implementation of very simple system and does not impose
-any significant limitations as all this data is meant for machine to
-machine communications and is not meant to be human readable.
+encoding allowed in {{!RFC8259}}. This simplifies implementation of very
```

Comment:
8428, from which this is stolen, says "a very simple system" (note the "a").  Which also is English :-)